ttfautohint/PKGBUILD

```diff
@@ -33,7 +36,7 @@ check() {
 package() {
   cd "$pkgname-$pkgver"
   make DESTDIR="$pkgdir" install
-  install -Dm644 -t "$pkgdir/usr/share/licenses/$pkgname/" COPYING FTL.TXT
+  install -Dm644 -t "$pkgdir/usr/share/licenses/$pkgname/" COPYING FTL.TXT GPLv2.TXT
```

Comment:
A copy of the GPL license does not need to be installed because the SPDX identifier matches a license installed by the licenses package.